guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Content Strategy K2.0 
+---
+
+## Table of contents
+
+- [Purpose and Audience](#purpose-and-audience)
+- [Information types](#information-types)
+  - [Concept Topics](#concept-topics)
+  - [Task (tutorial) topics](#task-tutorial-topics)
+  - [Reference topics](#reference-topics)
+  - [Troubleshooting topics](#troubleshooting-topics)
+  - [Release Notes](#release-notes)
+- [Graphics](#graphics)
+  - [Architecture and Flow Diagrams](#architecture-and-flow-diagrams)
+  - [Screenshots](#screenshots)
+- [Target Groups](#target-groups)
+  - [Decision maker](#decision-maker)
+  - [Software developer](#software-developer)
+  - [Admin/Operations](#adminoperations)
+- [Documentation Structure](#documentation-structure)
+  - [Overview](#overview)
+  - [Getting Started](#getting-started)
+  - [Deep Dive/Tutorials](#deep-divetutorials)
+  - [Operations](#operations)
+  - [Technical References](#technical-references)
+  - [Glossary](#glossary)
+- [Content Source](#content-source)
+
+## Purpose and Audience
+
+This content strategy focuses on the publicly available documentation under <https://kyma-project.io/docs/>.
+More documentation may be found in the Kyma Github repositories.
+
+The assumed reader of this guide has some basic knowledge of technical writing. To learn more, read the excellent [Istio document about adding documentation](https://istio.io/latest/docs/releases/contribute/add-content/).
+
+## Information types
+
+We follow a topic-based documentation approach, with one file per topic. Every documentation file has a clearly defined purpose, which is reflected in the title. The content must be able to stand on its own, but links should point to other documents as needed. 
+
+*For the structure within topics, I suggest to follow the DITA standard – because it’s pretty simple, it’s well-established, and it’s what is used in SKR documentation. Instead of currently 12-13 content types, we should thus be able to get away with just four (or five) types:*
+
+### Concept topics
+
+Answer "what-is" questions and provide essential background information that users must know.
+You'll find most concept topics in the Overview section, but they can be useful elsewhere too.
+
+Use nominal style for the title, for example, "Security" or "Security Concept".
+
+*Not sure we need much predefined structure here; it may depend on the content.*
+
+> [- DITA Concept Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-concept-topic.html)
```

Comment:
I just regard them as block quotes - we don't even need to keep them in the final version. I just wanted to add them so that we have a baseline to discuss what we want.

Reply:
removed them anyway :D